### PR TITLE
fix(aggregation-layers): Fix screen-grid-layer with CPU aggregation

### PR DESCRIPTION
**Pull request by donmccurdy**
- for https://github.com/visgl/deck.gl/issues/8318

Fixes examples/website/screen-grid, with support only for CPU-side aggregation currently.

Preview:

screengrid_nyc

### CODE REVIEW

**Comment by donmccurdy** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #8401** <a href="https://app.graphite.dev/github/pr/visgl/deck.gl/8401?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/visgl/deck.gl/8401?utm_source=stack-comment).

File: modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.ts [hunk 1]
```diff
@@ -152,8 +153,10 @@ export default class ScreenGridCellLayer<DataT = any, ExtraPropsT extends {} = {
           positions: new Float32Array([0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0])
         }
       }),
-      // @ts-expect-error TODO v9 API not as dynamic
-      isInstanced: true
+      bufferLayout: [
```
Comment (felixpalmer) [inline modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.ts:156]: Can you not do `bufferLayout: this.getAttributeManager()!.getBufferLayouts()`?

**Review by felixpalmer — APPROVED** [review]

**Review by ibgreen — APPROVED** [review]
Ship it

File: modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.ts (hunk 1 above)
Comment (donmccurdy) [reply modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.ts]: Ah that's better yes — thanks!

**Pull request merged**